README.md

```diff
@@ -88,41 +90,51 @@ class State(rx.State):
 
         self.processing, self.complete = True, False
         yield
-        response = openai.Image.create(prompt=self.prompt, n=1, size="1024x1024")
-        self.image_url = response["data"][0]["url"]
+        response = openai_client.images.generate(
+            prompt=self.prompt, n=1, size="1024x1024"
+        )
+        self.image_url = response.data[0].url
         self.processing, self.complete = False, True
-        
+
 
 def index():
     return rx.center(
         rx.vstack(
-            rx.heading("DALL·E"),
-            rx.input(placeholder="Enter a prompt", on_blur=State.set_prompt),
-            rx.button(
-                "Generate Image",
-                on_click=State.get_image,
-                is_loading=State.processing,
+            rx.heading("DALL-E", font_size="1.5em"),
+            rx.radix.text_field.root(
+                rx.input(
+                    placeholder="Enter a prompt..",
+                    on_blur=State.set_prompt,
+                    width="100%",
+                ),
```

Comment:
why is this wrapped in a root? maybe we can just set the width to some relative unit, like `width="25em"`?

Reply:
It has to because there is no way to affect the percentage width otherwise

Reply:
Ok I can do that for now